iptables/iptables.go

```diff
@@ -369,3 +376,19 @@ func ExistChain(chain string, table Table) bool {
 	}
 	return false
 }
+
+// GetVersion reads the iptables version numbers
+func GetVersion() (major, minor, micro int, err error) {
```

Comment:
True, just in case we want to do a version check in `Exists()`to decide whether attempt the `iptables -C ...` check instead of parsing the returned error string which I feel it is ugly.